src/XRP/protobuf-wrappers/index.ts

```diff
@@ -0,0 +1,27 @@
+export { default as XrpAccountDelete } from './xrp-account-delete'
```

Comment:
Does this not import correctly as `export XrpAccountSet from './xrp-account-set'`?

Reply:
hmm, y'know it actually doesn't seem to.  It gives me a "cannot find name XrpAccountSet" ...  Typescript Ted - can you weight in here?  am I doing these index files right?

Reply:
i think that's how it goes. afaict it replaces the need to do an import + an export.

https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export